config/.sass-lint.yml

```diff
@@ -0,0 +1,145 @@
+      convention: hyphenatedlowercase
+  hex-length:
+    - 1
+    - style: short
+  hex-notation: # TODO - Change this back to 1 later.
+    - 0
+    - style: lowercase
+  id-name-format: # TODO - Change this back to 1 later.
+    - 0
+    - convention: hyphenatedlowercase
+  indentation: # TODO - Change this back to 1 later.
+    - 0
+    - size: 2
+  leading-zero: # TODO - Change this back to 1 later.
+    - 0
+    - include: false
+  mixin-name-format:
+    - 1
+    - allow-leading-underscore: true
+      convention: hyphenatedlowercase
+  mixins-before-declarations: 1
+  nesting-depth: # TODO - Change this back to 1 later.
+    - 0
+    - max-depth: 3
+  no-color-keywords: 0 # TODO - Change this back to 1 later.
+  no-color-literals: 0 # TODO - Change this back to 1 later.
+  no-css-comments: 0 # TODO - Change this back to 1 later.
+  no-debug: 1
+  no-duplicate-properties: 0 # TODO - Change this back to 1 later.
+  no-empty-rulesets: 1
+  no-extends: 0
+  no-ids: 0 # TODO - Change this back to 1 later.
+  no-important: 0 # TODO - Change this back to 1 later.
+  no-invalid-hex: 1
+  no-mergeable-selectors: 0 # TODO - Change this back to 1 later.
+  no-misspelled-properties:
+    - 1
+    - extra-properties: []
+  no-qualifying-elements: # TODO - Change this back to 1 later.
+    - 0
+    - allow-element-with-attribute: false
+      allow-element-with-class: false
+      allow-element-with-id: false
+  no-trailing-zero: 0 # TODO - Change this back to 1 later.
+  no-transition-all: 0
+  no-url-protocols: 1
+  no-vendor-prefixes: # TODO - Change this back to 1 later.
+    - 0
+    - additional-identifiers: []
+      excluded-identifiers: []
+  placeholder-in-extend: 0 # TODO - Change this back to 1 later.
+  placeholder-name-format:
+    - 1
+    - convention: hyphenatedlowercase
+  property-sort-order:
+    - 0
+    - ignore-custom-properties: false
+  quotes:
+    - 1
+    - style: single
+  shorthand-values:
+    - 0
+  single-line-per-selector: 0 # TODO - Change this back to 1 later.
+  space-after-bang: # TODO - Change this back to 1 later.
+    - 0
+    - include: false
+  space-after-colon: # TODO - Change this back to 1 later.
+    - 0 
+    - include: true
+  space-after-comma: # TODO - Change this back to 1 later.
+    - 0
+    - include: true
+  space-before-bang: # TODO - Change this back to 1 later.
+    - 0
+    - include: true
+  space-before-brace: # TODO - Change this back to 1 later.
+    - 0
+    - include: true
+  space-before-colon: 0 # TODO - Change this back to 1 later.
+  space-between-parens: # TODO - Change this back to 1 later.
+    - 0
+    - include: false
+  trailing-semicolon: 0 # TODO - Change this back to 1 later.
+  url-quotes: 0 # TODO - Change this back to 1 later.
+  variable-for-property:
+    - 0
+    - properties: []
+  variable-name-format:
+    - 1
+    - allow-leading-underscore: true
+      convention: hyphenatedlowercase
+  zero-unit: 0 # TODO - Change this back to 1 later.
```

Comment:
this file shouldn't even really be here. We aren't using it so I am going to remove it.